opentelemetry-sdk/src/opentelemetry/sdk/metrics/_internal/aggregation.py

```diff
@@ -403,42 +401,43 @@ def __init__(
     ):
         super().__init__(attributes)
 
+        self._instrument_aggregation_temporality = (
```

Comment:
Rearranged these lines to improve consistency with `_ExponentialBucketHistogramAggregation`, this way it is much easier to compare them and find the analogous parts between them.